### PR TITLE
TST: new plots tests

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

Please check the type of change your PR introduces:
- [x] Other (please describe): Unit Tests

## Pull request checklist

Please check if your PR fulfills the following requirements, depending on the type of PR:

- Code base maintenance (refactoring, formatting, renaming):

  - [x] Docs have been reviewed and added / updated if needed
  - [x] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [x] All tests (`pytest --runslow`) have passed locally

## What is the current behavior?

We need unit tests to finish #287 development.

## What is the new behavior?

Tests were added, the coverage reported was generated and is printed below:
Obs.: The number 14, highlighted, should be, actually, 0.
image

To reproduce the report, just run: 
`coverage run --module pytest --runslow`
`coverage html`

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

@MateusStano , please read the commits descriptions 1 by 1 in order to understand what I've done.
Basically, a couple of MAINT were needed to optimize the tests, mainly, avoiding repetitions.
I am adding only two new tests, and I think everything is already being accomplished for now.
We can separate these two functions into more, if you want to. But I don't think it is necessary.

Ready for your review!

### CODE REVIEW

**Review by MateusStano — APPROVED** [review]
All good! Seems ready to merge
